examples/ollama/CommunityToolkit.Aspire.Hosting.Ollama.AppHost/Program.cs

```diff
@@ -1,13 +1,14 @@
 var builder = DistributedApplication.CreateBuilder(args);
 
 var ollama = builder.AddOllama("ollama")
-    .AddModel("phi3")
-    .WithDefaultModel("phi3")
+    .WithDefaultModel("phi:chat")
```

Comment:
As mentioned in a recent comment, let's remove the `WithDefaultModel` method and instead you always use `WithModel` and provide the model reference.